backend/editor/api.py

```diff
@@ -374,13 +360,17 @@ async def export_to_github(
 # Post methods
 
 
-@app.post("/{taxonomy_name}/{branch}/import", response_model=ImportFromGithubResponse)
+@app.post("/{taxonomy_name}/{branch}/import", response_model=models.ImportFromGithubResponse)
 async def import_from_github(
     request: Request,   
-    parameters: ImportFromGithubParameters
+    parameters: models.ImportFromGithubParameters,
 ):
     """
     Get taxonomy from Product Opener GitHub repository
+    
+    - **branch**: name of branch added by user
+    - **taxonomy**: name of the taxonomy added by user
+
```

Comment:
Hey @alexgarel 

I found this link: https://fastapi.tiangolo.com/tutorial/path-operation-configuration/?h=description#description-from-docstring

to add parameter description via docstring. 

But wasn't able to test it because http://api.taxonomy.localhost:8091/docs still gives 

```
Fetch error
Internal Server Error /openapi.json
```

Here are the logs: https://wtf.roflcopter.fr/paste/?b21a931c7f4e901b#CSVKdEb3GrBU4seKxRT8GhLMx52EoXy7Doj41MWiU6gA

[It points to line no. 90](https://github.com/openfoodfacts/taxonomy-editor/blob/main/backend/editor/api.py#L90) which has this code: 

```
        response = await call_next(request)
```

Reply:
I would prefer not to use this kind of documentation (which I find fragile, and we will have to repeat a lot of things) but rather really use models. See https://fastapi.tiangolo.com/tutorial/response-model/ and use dataclass or pydantic in models.py using field to add description etc.
(it seems for response you need to use dataclass rather than pydantic)

See https://fastapi.tiangolo.com/advanced/dataclasses/#dataclasses-in-nested-data-structures